tests/js/arraydemo.scrypttest.js

```diff
@@ -1,13 +1,16 @@
 const { expect, assert } = require('chai');
-const { buildContractClass, Ripemd160, Sig } = require('scryptlib');
+const { buildContractClass, Ripemd160, Sig, Bytes, buildTypeClasses } = require('scryptlib');
 const { compileContract } = require('../../helper');
 
+const compileResult = compileContract('arraydemo.scrypt');
+const ArrayDemo = buildContractClass(compileResult);
+const { ST1, ST2, ST3, MDArrayST1, AliasST2 } = buildTypeClasses(compileResult);
```

Comment:
Let's add `buildTypeClasses` into `scryptlib's` README, so ppl can know how to use it.